### ISSUE TITLE
LSC não consegue editar turma

### ISSUE THREAD

**Issue by gomeespedro** [alicerce-lab/alicerce-lab#87]
LSC que não está conseguindo editar turmas no LMS. 

https://web.telegram.org/k/stream/%7B%22dcId%22%3A1%2C%22location%22%3A%7B%22_%22%3A%22inputDocumentFileLocation%22%2C%22id%22%3A%224996841372519498516%22%2C%22access_hash%22%3A%22-2224078683806613784%22%2C%22file_reference%22%3A%5B1%2C0%2C0%2C76%2C227%2C101%2C176%2C253%2C53%2C43%2C233%2C8%2C225%2C32%2C97%2C149%2C91%2C38%2C220%2C244%2C255%2C204%2C93%2C187%2C90%5D%7D%2C%22size%22%3A38039685%2C%22mimeType%22%3A%22video%2Fmp4%22%2C%22fileName%22%3A%22LMS%20Alicerce%20-%20Google%20Chrome%202024-01-19%2018-34-14.mp4%22%7D

**Comment by gmfc** [comment]
Nao e possivel acessar o link

**Comment by gomeespedro** [comment]
https://github.com/alicerce-lab/alicerce-lab/assets/150803548/86a11e09-a291-4180-af46-404ce55017be

**Comment by gianpietrolucio** [comment]
@gomeespedro problema resolvido?

**Comment by gomeespedro** [comment]
Sim, fui informado que o acesso de LSC era limitado e que na verdade não dá pra editar turma
Então resolvido!
